### PR TITLE
41 create test database form

### PR DESCRIPTION
**Pull request by cvauclair**
# Issue NUMBER(ON WAFFLE)
Closes #ISSUENUMBER

#Changes in this pull request

#Screenshot or Gif of the FEATURE/THEME CHANGE/ OR VISUAL CHANGE

### CODE REVIEW

File: exampleShopifyApp/app.rb [hunk 1]
```diff
@@ -0,0 +1,170 @@
+require 'shopify_api'
```
Comment (evan01) [inline exampleShopifyApp/app.rb]: this should not be committed!

File: exampleShopifyApp/views/database_test_form.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<html>
```
Comment (evan01) [inline exampleShopifyApp/views/database_test_form.erb]: exampleShopifyApp/* should also not be merged

**Review by evan01 — CHANGES_REQUESTED** [review]
Please attach a picture/gif showing what your changes look like. Also make sure to reference the waffle issue you are closing. Looks great though!

**Comment by cvauclair** [comment]
Added screenshots of the admin urls
screenshot from 2018-03-11 15-11-57
screenshot from 2018-03-11 15-11-53
screenshot from 2018-03-11 15-11-45
screenshot from 2018-03-11 15-11-36
screenshot from 2018-03-11 15-11-29
screenshot from 2018-03-11 15-10-24

**Comment by luantran** [comment]
Is it that necessary to have to add the Lat and Long coordinates? It doesn't feel user-friendly...I feel like the user should only have to add an address, and then we can either process that through an API before inserting into the DB. Or I could also process it after the GET request (not ideal tho)

**Review by evan01 — APPROVED** [review]

**Pull request merged**